lib/firebase_options.dart

```diff
@@ -0,0 +1,69 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      throw UnsupportedError(
+        'DefaultFirebaseOptions have not been configured for web - '
+        'you can reconfigure this by running the FlutterFire CLI again.',
+      );
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for macos - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.linux:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for linux - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions android = FirebaseOptions(
+    apiKey: 'AIzaSyDA0KXXjHLvjxQb7c8RHArPnYDAy546-aI',
+    appId: '1:791091683642:android:26c4a5ab95f1ccb99ab003',
+    messagingSenderId: '791091683642',
+    projectId: 'etsmobile-14206',
+    storageBucket: 'etsmobile-14206.appspot.com',
+  );
+
+  static const FirebaseOptions ios = FirebaseOptions(
+    apiKey: 'AIzaSyD8i_iLCb2yZ1XZ1YpBTSxqMT1Y1ZVKMGY',
```

Comment:
Est ce qu'ont veux vraiment expose la key?

Reply:
J'pense que on est correcte de le laisser, c'est un fichier auto généré. Mais on peut regarder ce que ça implique plus en profondeur.
![image](https://user-images.githubusercontent.com/25663435/197865604-01a2e2bd-41df-4b89-885d-74e5ddaf937c.png)

Reply:
https://stackoverflow.com/questions/71432691/flutter-should-firebase-options-dart-be-added-to-gitignore